contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2870,15 +2870,30 @@ static void process_select_statement(
 ANTLR_result
 antlr_parser_cpp(const char *sourceText)
 {
-	ANTLR_result result;
+	ANTLR_result result = {
```

Comment:
This doesn't look good to me, we don't need to initialize every member of struct every time.